oddbird/oddleventy
# Disable indents parsed as md code blocks

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![](https://images.unsplash.com/photo-1596767317930-ff649915e589?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=900&q=60)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/aPm5jIYS/92-prevent-mistakenly-rendering-html-nunjucks-in-markdown-as-indented-code-blocks

## Description
_The commit messages say what you did; this should explain why and/or how._



## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

1. open `embed.macros.njk`
2. add empty lines before `media-image` and `media-content` (lines 54 & 55)
3. (for testing on `main`, you will also have to indent those lines with 4 spaces)
4. look at any page with quotes (scroll down the home page)

That should break the quote images on `main` branch, but not on this branch…

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screen Shot 2020-08-10 at 11 23 35 AM](https://user-images.githubusercontent.com/104161/89811817-70f5eb80-dafc-11ea-9453-b036d991ae94.jpg)
![Screen Shot 2020-08-10 at 11 25 58 AM](https://user-images.githubusercontent.com/104161/89811825-73584580-dafc-11ea-883c-3c348fbe94cc.jpg)
